src/components/atoms/Icon.tsx

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright 2022 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from "react";
+
+import liveIcon from "../../../res/img/element-icons/live.svg";
+
+export enum IconType {
+    Live,
+}
+
+const iconTypeMap = new Map([
+    [IconType.Live, liveIcon],
+]);
+
+export enum IconColour {
+    Accent = "accent",
+    White = "white",
```

Comment:
How does this work with themes?

Reply:
How can we do this? Adding something like `$alertText` or `$alertColor` that can be applied to text where the background is `$alert`. Or by having dedicated colours for the live badge: `$liveBadgeBackground` and `$liveBadgeColour`.

Reply:
Should it not just use `primary-content` or `background`?

Reply:
`primary-content` in light theme is black. That is different from the Voice Broadcast designs. `background` is the overall app background I think. Works in light mode. Doesn't match in dark mode. :shrug: 

I can put this on my list for our next sync with Gaëlle and will use a specialised colour for now.